jigna/tests/test_jigna_vue_web.py

```diff
@@ -42,11 +43,17 @@ def setUpClass(cls, async=False):
         t.setDaemon(True)
         t.start()
 
-        browser = webdriver.Firefox()
+        if sys.platform.startswith('darwin'):
```

Comment:
Can you please add a comment describing the reason for switching to Chrome for OSX and maybe a link to the issue which says FF>45 doesn't work with selenium? Thanks.